components/site/Masthead/MastheadContainer.js

```diff
@@ -4,7 +4,17 @@ import { Box } from '@twilio-paste/core/box';
 
 export const MastheadContainer = ({ children }) => {
   return (
-    <Box as="header" backgroundColor="colorBackgroundPrimaryDarkest" color="colorTextInverse" position="sticky" top="0">
+    <Box
+      as="header"
+      backgroundColor="colorBackgroundPrimaryDarkest"
+      color="colorTextInverse"
+      position="sticky"
+      top="0"
+      zIndex="2000"
```

Comment:
Let's make this one `zIndex20` to use one of our tokens

Reply:
Done.